TriblerGUI/widgets/lazytableview.py

```diff
@@ -82,7 +82,7 @@ def on_subscribe_control_clicked(self, index):
         item = index.model().data_items[index.row()]
         # skip LEGACY entries, regular torrents and personal channel
         if (u'subscribed' not in item or
-                item[u'status'] == 6 or
+                item[u'status'] == 1000 or
```

Comment:
We should re-use the metadata state definitions here.

Reply:
But I guess you were against adding dependencies to TrilberGUI from the Core, right?

Reply:
Hm. I'll just add them to `defs.py`.

Reply:
Yes but at this point I rather have a link to the core (which we can later refactor) instead of having a magic variable which is unclear.